ci-operator/jobs/MERGE_BLOCKING_STATUS_MIGRATION.md

```diff
@@ -0,0 +1,24 @@
+## Merge Blocking
+By making a change to the test configuration that adds, removes or renames a merge-blocking status will result in a merge queue that is stuck unless manual steps with the [migratestatus](https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus) tool are taken.
```

Comment:
_"By making a change to the test configuration that ... will result..."_ does not sound correct English, probably an editing artifact. 

Either _"Making a change ... will result..."_ or even simplifying the whole sentence to _"Adding, removing or renaming merge-blocking status in a test configuration may result in..."_

Writing this, I realized the sentence is unnecessary strict - just changing config may be totally safe, it is only risky when there are in-flight PRs while the change is made. The doc should mention this.